entry_types/scrolled/package/src/entryState/structure.js

```diff
@@ -105,12 +104,29 @@ export function useSectionContentElements({sectionId}) {
 
 export function useChapters() {
   const chapters = useEntryStateCollectionItems('chapters');
-
+  let chapterSlugs = {};
   return chapters.map(chapter => {
+    let chapterSlug = chapter.configuration.title;
+    if (chapterSlug) {
+      chapterSlug = slugify(chapterSlug, {
+        lower: true,
+        locale: I18n.locale,
+        strict: true
```

Comment:
Passing `'de'` here makes it replace umlauts for me. `I18n.locale` is not save to use in React components, since it's value depends on the environment (editor, published entry etc.). That's why we have hooks like `useLocale`/`useI18n`. But since Slugify only supports one other locale besides `de` which in turn is not supported by Pageflow, I suggest to hard code `'de'` here. The other replacements (like $ -> dollar) are a bit weird, but inline with what I'd expect after looking at the Slugify source.

Reply:
weird for me it did not work by hard coding 'de' as well.

Reply:
Check your locally installed slugify package. I had to run `yarn install` to update it since, before your commit, there already was another package depending on an older version which did not yet include de locale.